### PR TITLE
Update inventory-configure-distance-priority.md

### PR DESCRIPTION
**Pull request by agorbivskyi**
## Purpose of this pull request

Add critical information on how to configure "In-store Delivery" shipping method

## Magento release version

2.4.0

## Affected documentation pages

https://docs.magento.com/user-guide/shipping/shipping-in-store-delivery.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/shipping/shipping-in-store-delivery.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ With the in-store delivery method, the customer can select a source to be used a
 ![]({% link images/images/luma-in-store-example.png %}){: .zoom}
 [_In-store Delivery Method at Checkout_]({% link sales/checkout-step1-shipping.md %})
 
+{:.bs-callout-warning}
```
Comment (leslietilling) [inline src/shipping/shipping-in-store-delivery.md:10]: We will have to use a div to create a bullet list in a warning note.

```suggestion
<div class="bs-callout-warning" markdown="1">
```

File: src/shipping/shipping-in-store-delivery.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ With the in-store delivery method, the customer can select a source to be used a
...
+**Before setup:**</br>
+
+- Make sure you have a non-default stock and source. For more information about how to configure a source as a pickup location, see [Adding a New Source]({% link catalog/inventory-sources-add.md %}).
+
+- Make sure you have configured a Distance Priority Algorithm, see [Configuring Distance Priority Algorithm Edit on GitHub Log an Issue]({% link catalog/inventory-configure-distance-priority.md %}).
+
```
Comment (leslietilling) [inline src/shipping/shipping-in-store-delivery.md:16]: Close the div

```suggestion

</div>
```

**Comment by leslietilling** [comment]
running tests

**Comment by leslietilling** [comment]
@agorbivskyi  Please update with master. It looks like settings for your fork branch will not allow me to do that and I cannot merge without.

**Comment by agorbivskyi** [comment]
@leslietilling done.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**